_data/simple-icons.json

```diff
@@ -13530,6 +13530,11 @@
             "hex": "0FAAFF",
             "source": "https://www.sap.com"
         },
+        {
+            "title": "Sartorius",
+            "hex": "000000",
```

Comment:
I would argue the 'brand color' is `#FFED00` - it's used extensively throughout the site and socials. What do you think @CodeShakingSheep?

Reply:
I'd agree with using `#ffed00`.

Reply:
I assumed that the brand color from the logo should be used. But yes, the yellow color makes sense. I just updated it.